addon/components/paper-switch.js

```diff
@@ -1,3 +1,5 @@
+/* globals Hammer */
```

Comment:
@alexlafroscia HammerJS is AMD compliant. I think the way to go would be to import it like [this](http://www.ember-cli.com/managing-dependencies/#standard-amd-asset). Meanwhile, I think explicitly adding `/* globals Hammer */` is a reminder for us to do it. Later we just replace it with an ES6 `import`.